src/genie/libs/parser/viptela/show_control_local_properties.py

```diff
@@ -0,0 +1,453 @@
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any
+from pprint import pprint
+import re
+
+# ===============================================
+# Schema for 'show control local-properties'
+# ===============================================
+
+class ShowControlLocalPropertiesSchema(MetaParser):
+
+    """ Schema for "show control local-properties" command """
+
+    schema = {
+        "personality": str,
+        "sp_organization_name": str,
+        "organization_name": str,
+        "root_ca_chain_status": str,
+        "certificate_status": str,
+        "certificate_validity": str,
+        "certificate_not_valid_before": str,
+        "certificate_not_valid_after": str,
+        "enterprise_cert_status": str,
+        "enterprise_cert_validity": str,
+        "enterprise_cert_not_valid_before": str,
+        "enterprise_cert_not_valid_after": str,
```

Comment:
these could be Optional
```suggestion
        Optional("enterprise_cert_status"): str,
        Optional("enterprise_cert_validity"): str,
        Optional("enterprise_cert_not_valid_before"): str,
        Optional("enterprise_cert_not_valid_after"): str,
```